obs-tools/opensuse42_submit.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+LIST=`osc list openSUSE:Factory | grep -i -e yast2 -e skelcd -e "^libstorage$" -e perl-bootloader -e linuxrc`
+
+for i in $LIST; do
+  # compare if something is new by comparing binaries ( as version is part of rpm output )
+  OS42=$(osc list -b openSUSE:42 $i -r standard -a x86_64)
+  FACTORY=$(osc list -b openSUSE:Factory $i -r standard -a x86_64)
+  if [ "$OS42" == "$FACTORY" ]; then
+     continue
+  fi
+  echo "submitting $i..."
+  osc sr openSUSE:Factory $i openSUSE:42 -m "opensuse42_submit.sh: update of yast2 related package to the SLE12SP1 version"
```

Comment:
Why mention SLE12SP1 when it's really Factory? It's true that we are trying to keep both in sync for the time being. But just imagine the case in which a request is accepted into Factory but not yet into SP1, then the message will not be accurate.

Going a little bit too far, we could even have a SOURCE="openSUSE:Factory" variable to use along the script. To keep the commands and the message in sync.